### PR TITLE
Create confirmation status alert (the last page in the preview send flow) #1211

### PR DESCRIPTION
**Pull request by heyitsmebev**
Issue ticket: 
https://github.com/GSA/notifications-admin/issues/1211

Figma design: 
https://www.figma.com/file/aAWhlPzOP6kfVFnCCAfSZZ/notify.gov?type=design&node-id=2233-2998&mode=design&t=E9xyph1T7fWIrFNU-0

Discussions and notes around what it should look like: https://github.com/GSA/notifications-admin/issues/815#issuecomment-1960331537

## Description

When the user sends or schedule a message, there should be a confirmation alert and details about the message status

(when scheduled)
image(when sent)
image
## What's considered done
- [x]  display "human" readable date within the alert banner
- [x] There should be an alert banner for "send" (green banner) and "schedule" (blue banner) in the following format:
H4: Your text has been scheduled
Body: [Template name] from texting service was scheduled [date] [time]
- [x] Update page title to "Message status"
- [x] Add in Message preview, Recipient list, and SMS status headers 
- [x] This message status should only be available if a user is arriving to that page from the "preview page" within the send flow or the job is still processing 
- [x] Whether the user is send a message via one-offs or upload csv, the user should be redirected to the view_job page. This address this issue here: https://github.com/GSA/notifications-admin/issues/1149

### CODE REVIEW

File: app/formatters.py [hunk 1]
```diff
@@ -98,6 +98,13 @@ def format_datetime_short(date):
     )
 
 
+def format_datetime_short_12h(date):
+    # example: February 20, 2024 at 07:00 PM US/Eastern
+    return "{} at {} {}".format(
```
Comment (terrazoon) [inline app/formatters.py:103]: I think I'm getting a little confused about how many datetime formats are needed for our app.  I know @maomeara63 wrote a ticket that caused me to change all the 'format_datetime_short' formats to 'format_datetime_short_america', but now 'format_datetime_short' is coming back in a new form.  Do we really want dates in different formats in different places?  I would prefer UTC 24 hour format everywhere but I know I'm the only one who wants that. :)

**Review by terrazoon — COMMENTED** [review]
I don't understand a couple things:

1. Why are we adding format_datetime_short_12_hour instead of just changing format_datetime_short?  I'm the only person who likes 24 hour and everyone else wants 12 hour everywhere, so why add a new formatter instead of fixing the old?

2. Why do we want format_datetime_short_12_hour instead of format_datetime_short_america?  Shouldn't we standardize on one datetime format, period, for the whole UI?

https://gsa-tts.slack.com/archives/C0107UC0W1M/p1709576714469049
> According to the style guide, the date should be spelled out in text and numerical in a table. I'm not a fan of the leading zeros, but I can live with that.

I can create an issue ticket where we fix the dates from format_datetime_short-> format_datetime_table. That way, we standardize the dates with the spelled out text and numerical for tables 

03-01-2024 at 01:48 PM
February 20, 2024 at 07:00 PM US/Eastern

File: app/formatters.py (hunk 1 above)
Comment (heyitsmebev) [reply app/formatters.py:103]: @terrazoon Good point.

So far, these are the two date formats that have been connected with the redesign. I would be okay if we standardized the dates to fit these formats as well or just one format 

03-01-2024 at 01:48 PM
February 20, 2024 at 07:00 PM US/Eastern

**Comment by ccostino** [comment]
Thanks, @heyitsmebev!  It looks like this should be good to go now, but would you mind updating it one more time against `main` please, just to make sure?  A few things have made their way in over the last couple of days and I want to make sure these changes still work as expected.  Thanks!

**Review by ccostino — APPROVED** [review]
Thanks, @heyitsmebev!

**Pull request merged**